settings.gradle

```
@@ -5,3 +5,4 @@ include ':base'
 include ':messagingpush'
 include ':common-test'
 include ':messaginginapp'
+include ':kotlin_compose'
```

Does this show up in Android project view if you open the SDK? Asking because it is placed in `samples` folder, and without specifying path I think it might not show sample code when opening the SDK.

I suggest updating as follows so we can group all sample apps inside sample directory in Android project view when opening the SDK in Android Studio

```suggestion
include ':samples:kotlin_compose'
```